crates/router/src/db/address.rs

```diff
@@ -70,15 +107,18 @@ impl AddressInterface for Store {
             .await
     }
 
-    #[instrument(skip_all)]
-    async fn update_address(
+    async fn insert_address_payments(
         &self,
-        address_id: String,
-        address: storage::AddressUpdate,
+        _payment_id: &str,
```

Comment:
If address has payment_id already present in the object, don't pass payment_id here.

Reply:
Even though address has payment_id, it's optional value. But while inserting address for payments we need payment_id as mandatory field!

Reply:
if this is mandatory one then you to use here too.

Here discarding the payment_id and we are saying mandatory won't give full guarantees.

Reply:
This is mandatory while we are using kv for insert_address_payments, this is done in next PR